worlds/noita/__init__.py

```diff
@@ -0,0 +1,58 @@
+from BaseClasses import Item, Tutorial
+from worlds.AutoWorld import WebWorld, World
+from . import Events, Items, Locations, Options, Regions, Rules
+
+
+class NoitaWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Noita integration for Archipelago multiworld games.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Heinermann", "ScipioWright", "DaftBrit"]
+    )]
+    theme = "partyTime"
+    bug_report_page = "https://github.com/DaftBrit/NoitaArchipelago/issues"
+
+
+# Keeping World slim so that it's easier to comprehend
+class NoitaWorld(World):
+    """
+    Noita is a magical action roguelite set in a world where every pixel is physically simulated. Fight, explore, melt,
+    burn, freeze, and evaporate your way through the procedurally generated world using wands you've created yourself.
+    """
+
+    game = "Noita"
+    option_definitions = Options.noita_options
+    topology_present = True
+
```

Comment:
Considering you want to get this merged, let's also add `data_version = 1` so the Data Package isn't in custom mode anymore.

```suggestion
    data_version = 1
```